From ce34244c9a5739910b9ff81007d5d094a801bca4 Mon Sep 17 00:00:00 2001
From: Benoit Gschwind <gschwind@gnu-log.net>
Date: Sun, 12 Jun 2016 11:07:25 +0200
Subject: [PATCH] fix the xcb-shape configuration flags

Fix configure.ac and Makefile.am to check and use xcb-shape properly.
---
 configure.ac    | 2 +-
 src/Makefile.am | 2 ++
 2 files changed, 3 insertions(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index b91f35d..c535d27 100644
--- a/configure.ac
+++ b/configure.ac
@@ -45,7 +45,7 @@ PKG_CHECK_MODULES(XCBRR, xcb-randr >= 1.0.0)
 AC_SUBST(XCBRR_CFLAGS)
 AC_SUBST(XCBRR_LIBS)
 
-PKG_CHECK_MODULES(XCBDMG, xcb-shape >= 1.0.0)
+PKG_CHECK_MODULES(XCBSHP, xcb-shape >= 1.0.0)
 AC_SUBST(XCBSHP_CFLAGS)
 AC_SUBST(XCBSHP_LIBS)
 
diff --git a/src/Makefile.am b/src/Makefile.am
index e928274..7f394a3 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -9,6 +9,7 @@ AM_CXXFLAGS =  \
 	$(XCBUTIL_CFLAGS) \
 	$(XCBFIXES_CFLAGS) \
 	$(XCBDMG_CFLAGS) \
+	$(XCBSHP_CFLAGS) \
 	$(XCBRR_CFLAGS) \
 	$(XCBCOMP_CFLAGS) \
 	$(CAIRO_CFLAGS) \
@@ -48,6 +49,7 @@ page_LDADD = \
 	$(XCBUTIL_LIBS) \
 	$(XCBFIXES_LIBS) \
 	$(XCBDMG_LIBS) \
+	$(XCBSHP_LIBS) \
 	$(XCBRR_LIBS) \
 	$(XCBCOMP_LIBS) \
 	$(CAIRO_LIBS) \
-- 
2.7.3

